ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml

```diff
@@ -0,0 +1,29 @@
+base_images:
+  tools:
+    name: "4.14"
+    namespace: ocp
+    tag: tools
+build_root:
+  from_repository: true
```

Comment:
@jianzhangbjz should not use it this way because there is no definition in repo.
it is why you meet the issue
```console
  * failed to generate steps from config: failed to get steps from configuration: failed to read buildRootImageStream from repository: failed to read .ci-operator.yaml file: open .ci-operator.yaml: no such file or directory 
```
in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/39619/rehearse-39619-periodic-ci-openshift-release-tests-master-stable-build-test/1661287588591308800

FYI https://docs.ci.openshift.org/docs/architecture/ci-operator/#build-root-image

Reply:
Yes, thanks!